ir/ir.def

```diff
@@ -226,8 +230,8 @@ class Entry : IAnnotated {
 
 /// List of predefined entries. Part of table properties
 class EntriesList : PropertyValue {
-    Vector<Entry> entries;
-    size_t size() const { return entries->size(); }
+    inline Vector<Entry> entries;
```

Comment:
There are still some non-inline Vectors.  I think this does make all the non-inlined IndexedVectors go away, so it should be possible to get rid of IndexedVector and replace it with something simpler (NameMap?)